### DISCUSSION TITLE
22621.1413 keeps crashing and never recovers?

### DISCUSSION THREAD

**Discussion by surajkrishnan14** [valinet/ExplorerPatcher#1204] [General]
Are there known issues with recent versions of explorer patcher and 22621.1413 OS build (Windows 11 enterprise)? I installed it on a machine today and explorer.exe seems to crash continuously and never recovers. The start menu and windows keys never work, eventually I had to use task manager to run cmd and uninstall explorer patcher.

**Comment by hoffy5000** [comment]
same issue here, I did the sfc and dism stuff.

I noticed this though when I start explorer with command line:

[Updates] No updates are available.
[Symbols] Started "Download symbols" thread.
[Symbols] Attempting to download symbols for unknown OS version 10.0.22621.1483.
[Symbols] Downloading to "C:\Users\rshof\AppData\Roaming\ExplorerPatcher\".
[Symbols] Downloading symbols for "C:\WINDOWS\system32\twinui.pcshell.dll" ("71c636f7f27bb03d446e8d6696ad3a7b")...
[Symbols] Reading symbols...
[Symbols] Windows 10 window switcher style is not available in this build.
[Symbols] Failure in reading symbols for "C:\WINDOWS\system32\twinui.pcshell.dll".

eventvwr shows that dll as the faulting module in explorer.exe crashing
